### DISCUSSION TITLE
Lawn Mowing

### DISCUSSION THREAD

**Discussion by Ploni7** [OpenRCT2/OpenRCT2#17318] [Q&A] [answered]
How do you enable the handymen to have mow lawns automatically selected?

**Comment by telk5093** [comment]
Not available for now, maybe plugin's API can do this.

**Reply by Ploni7** [reply]
Forgive my ignorance what is plugin's API exactly?

**Reply by Broxzier** [reply]
An API exposes all features that plug-ins can use, as shown here: https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/openrct2.d.ts

~~There is currently no support for changing a staff's chores.~~

There is the action "staffsetorders" which you can use from plug-ins, so it is possible. A plug-in then has to detect when a new handyman is hired and use this action accordingly.

**Reply by Ploni7** [reply]
How do I use the file?

**Reply by Broxzier** [reply]
https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/scripting.md#writing-scripts You need some experience with writing code. Someone who knows how to may create a plug-in for this.

**Reply by telk5093** [reply]
It should that someone make a plugin that supports such feature.

**Reply by Ploni7** [reply]
Can someone make a plugin that supports such a feature or tell me how to do it?

**Reply by IntelOrca** [reply]
@Ploni7 you could try asking on \#plugin on the discord server. Some people have funded a plugin that they needed, similar to a bug bounty.

**Comment by ageekhere** [comment]
Does Lawn Mowing do anything or is it just visual? Would be nice if it had more of an impact in the game.

**Reply by ocalhoun6** [reply]
It's only visual for now.

Not sure that should be changed, either. Because not every park would look better with mowed grass -- some parks are meant to look forested and unkempt ... but if there was an advantage to mowing or a disadvantage to not mowing, then you'd feel compelled to mow the forest.

It could also be placing an undue influence on the player's choice of surface to use.
- If there's an advantage to mowed grass, then that incentivizes players to choose grass so that it can be mowed.
- If there's a disadvantage to unmowed grass, then that incentivizes players to choose other surface types so they don't have to assign handymen to mow that grass.

------

Personally, I think it's better left as aesthetic only. Because if you make it affect gameplay, then optimized parks would pretty much _have_ to use it, making it a practical choice rather than a creative choice.

(And changing it could also affect the balance of some scenarios. By making grassy scenarios easier or more difficult than non-grassy scenarios.)

**Reply by 733737** [reply]
Just add a new staff type the landscaper that waters plants and mows grass, and remove those from handymen

**Comment by StrikerStorm** [comment, marked as answer]
Navigate to the C:\Users\Your Name\Documents\OpenRCT2 folder, and open up 'config.ini' with notepad. Under the [general] section look for 'handymen_mow_default' and set the value to true!

**Reply by Broxzier** [reply]
config.ini

**Reply by StrikerStorm** [reply]
My bad, edited to change it

**Comment by Ploni7** [comment]
THANK YOU StrikerStorm for the answer.
